Source/SwiftLintFramework/Rules/Idiomatic/ToggleBoolRule.swift

```diff
@@ -11,7 +11,6 @@ public struct ToggleBoolRule: SubstitutionCorrectableRule, ConfigurationProvider
         name: "Toggle Bool",
         description: "Prefer `someBool.toggle()` over `someBool = !someBool`.",
         kind: .idiomatic,
-        minSwiftVersion: .fourDotTwo,
```

Comment:
maybe this should be enabled by default now?

Reply:
decided not to since it relies on regexes pretty heavily and I wouldn't be too surprised if there are false positives out there